source/PlayerInfo.cpp

```diff
@@ -2016,6 +2018,47 @@ void PlayerInfo::SetTravelDestination(const Planet *planet)
 
 
 
+void PlayerInfo::QueueTeleport(const Planet *teleportPlanet)
+{
+	teleportDestination = teleportPlanet;
+}
+
+
+
+void PlayerInfo::DoQueuedTeleport()
+{
+	if(teleportDestination)
+	{
+		system = teleportDestination->GetSystem();
+		planet = teleportDestination;
+		flagship->SetSystem(system);
+		flagship->SetPlanet(planet);
+		for(const shared_ptr<Ship> &ship : ships)
```

Comment:
You teleport every single ship that the player owns? Even parked ones?

Reply:
I will fix this

Reply:
Fixed